src/templates/zh-CHT/base.html

```diff
@@ -0,0 +1,188 @@
+{% block methodology_stat_1 %}5.8 百萬{% endblock %}
+{% block methodology_stat_2_title %}數據處理數量Data Processed{% endblock %}
+{% block methodology_stat_2 %}20.9 兆位元組{% endblock %}
+{% block methodology_description %}
+除另有備註外，Web Almanac 網路年鑑20個章節的統計指標皆萃取自HTTP Archive數據集。HTTP Archive是社群推動的專案，自2010年開始持續追蹤網路如何建置，每月透過WebPageTest和Lighthouse測試將近6百萬個網站並將資料集公開置於BigQuery供眾分析。Web Almanac網路年鑑的統計指標採用2019年7月的數據集為基準，更多的資訊請參閱統計方法頁面。
+{% endblock %}
+{% block methodology_link %}瞭解統計方法{% endblock %}
+
+{% block introduction %}介紹{% endblock %}
+
+{% block foreword_title %}前言{% endblock %}
+
+{% block foreword %}
+    <p>
+      當你踏進科技網絡世界後會發現開放的網路是如此撲朔迷離的美麗，透過緊密相連的網路生態系統讓擁抱網路的產業與職業共創成功。我們瞭解網路很重要，但卻難以捉摸網路究竟如何運行。自從2010年，HTTP Archive啟動這個迷人的專案，開始持續追蹤網路如何建構。然而還有一件使命深深吸引著Web Almanac網路年鑑的挑戰：賦予這些數據集生命力並促使社群能更容易瞭解網路究竟如何運行，這也是Web Almanac網路年鑑熱情傳播的所在。
+    </p>
+
+    <p>
+      Web Almanac網路年鑑的任務是將原本只有數據挖礦勇者才能發掘的寶藏公諸大眾並將其整理為簡單易懂的年鑑，這是產業專家們熱情的挹注才讓數據重燃生命展現脈絡。Web Almanac的20個章節分別展現網路的各種脈絡，其由各個領域的專家作者撰寫後再由夥伴互相審稿後誕生，Web Almanac網路年鑑的養分是由撰寫撫育它茁壯的專家的經驗知識堆疊。
+    </p>
+
+    <p>
+      Web Almanac網路年鑑收穫許多值得慶祝的里程碑，但它也提醒我們要更精益求的推動高品質的使用者體驗。透過各個章節的數據脈絡分析讓我們更勇於承擔使命，並分享如何開發更好的網路。這就像是為迷途羞愧的羔羊點燃熊熊烈火指引明亮的康莊大道，透過網路社群不斷挹注養分的推動下，我們希望將統計年鑑視為年度傳統，讓每年度能視需要追蹤進度，並重燃照耀康莊大道。
+    </p>
+
+    <p>
+      你將會發現這本年鑑有許多值得學習的地方，現在就啟程探索並向社群分享你的想法，這將讓我們可以瞭解使用者閱讀網路年鑑的體驗。
+    </p>
+
+    <p>— <em><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}">Rick Viscomi</a>, creator of the Web Almanac</em></p>
+{% endblock %}
+
+{% block coming_soon %}即將來到!{% endblock %}
+
+{% block appendix %}附錄{% endblock %}
+{% block appendices %}附錄{% endblock %}
+
+{% block ebook_title %}電子書{% endblock %}
+{% block ebook_download %}下載 {{ year }} Web Almanac 網路年鑑： PDF 格式 ({{ ebook_size_in_mb }}MB){% endblock %}
+{% block ebook_download_note %}(使用 <a href="https://www.princexml.com/">www.princexml.com</a>){% endblock %}
+
+{% 
+  set localizedPartTitles = {
+    "I. Page Content": "I. 頁面內容",
+    "II. User Experience": "II. 使用者體驗",
+    "III. Content Publishing": "III. 內容發布",
+    "IV. Content Distribution": "IV. 內容傳遞"
+  }
+%}
+{%
+  set localizedChapterTitles = {
+    "javascript": "JavaScript",
+    "css": "CSS",
+    "markup": "標記",
+    "media": "媒體",
+    "third-parties": "第三方",
+    "fonts": "字體",
+    "performance": "效能",
+    "security": "安全",
+    "accessibility": "無障礙網頁",
+    "seo": "搜尋引擎優化",
+    "pwa": "漸進式網頁應用程式",
+    "mobile-web": "行動裝置網頁",
+    "ecommerce": "電子商務",
+    "cms": "內容管理系統",
+    "compression": "壓縮",
+    "caching": "緩存",
+    "cdn": "內容傳遞網路",
+    "page-weight": "網頁大小",
+    "resource-hints": "資源提示",
+    "http2": "超文字傳輸協定/第2版",
+    "privacy": "隱私",
+    "capabilities": "功能",
+    "jamstack": "Jamstack"
+  }
+%}
+
+{% set localizedTeamNames = {
+  "analysts": "分析者",
+  "authors": "作者",
+  "brainstormers": "腦力激盪者",
+  "designers": "設計者",
+  "developers": "開發者",
+  "editors": "審稿者",
+  "leads": "專案主導者",
+  "reviewers": "審稿者",
+  "translators": "翻譯者", 
+  }
+%}
```

Comment:
Nit but can we ensure all files end in a blank line:

```suggestion
%}

```

Reply:
thanks to the reminder, and as curiosity would you talk a little bit about what efforts will this blank line bring us.

Reply:
It can cause some issues in certain processing: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Not seen it causing a real issue to be honest but I’m cautious. Plus the red mark when you review the file just annoys me!! 😀